pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -1723,6 +1723,97 @@ var _ = Describe("Converter", func() {
 			Expect(domain.Spec.Devices.Interfaces[0].Type).To(Equal("ethernet"))
 			Expect(domain.Spec.Devices.Interfaces[1].Type).To(Equal("ethernet"))
 		})
+		It("Should create network configuration for macvtap interface and a multus network", func() {
+			v1.SetObjectDefaults_VirtualMachineInstance(vmi)
+			multusNetworkName := "multusNet"
+			networkName := "net1"
+
+			iface1 := v1.Interface{Name: networkName, InterfaceBindingMethod: v1.InterfaceBindingMethod{Macvtap: &v1.InterfaceMacvtap{}}}
+
+			multusNetwork := v1.Network{
+				Name: networkName,
+				NetworkSource: v1.NetworkSource{
+					Multus: &v1.MultusNetwork{NetworkName: multusNetworkName},
+				},
+			}
+			vmi.Spec.Networks = []v1.Network{multusNetwork}
+			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{iface1}
+
+			domain := vmiToDomain(vmi, c)
+			Expect(domain).NotTo(BeNil())
```

Comment:
well the point is to add something that will help your future self/developer debugging why this test fail - to find exactly what failed and where.
surely adding "should not be nil" is not that informative, but if it would be something like "domain should not be nil after rendering phase" (made something up) then it would be more helpful..

Reply:
+ it's a good documentation to how things should work..